boards/QUARTER_BOARD/board.mk

```diff
@@ -0,0 +1,5 @@
+# List of all the board related files.
+BOARDSRC = /home/meli/ROBOCUP/boards/QUARTER_BOARD/board.c

```

Comment:
Would it be possible to make these paths not hard-coded? I.e. either change them to be relative (and test, whether it works), or find another way, because we can't have hard-coded paths like this in the code, it just removes the whole share-ability.